internal/builder/builder.go

```diff
@@ -64,21 +64,22 @@ const (
 
 // Builder represents a pack builder, used to build images
 type Builder struct {
-	baseImageName        string
-	image                imgutil.Image
-	layerWriterFactory   archive.TarWriterFactory
-	lifecycle            Lifecycle
-	lifecycleDescriptor  LifecycleDescriptor
-	additionalBuildpacks []buildpack.BuildModule
-	additionalExtensions []buildpack.BuildModule
-	metadata             Metadata
-	mixins               []string
-	env                  map[string]string
-	uid, gid             int
-	StackID              string
-	replaceOrder         bool
-	order                dist.Order
-	orderExtensions      dist.Order
+	baseImageName            string
+	image                    imgutil.Image
+	layerWriterFactory       archive.TarWriterFactory
+	lifecycle                Lifecycle
+	lifecycleDescriptor      LifecycleDescriptor
+	additionalBuildpacks     buildpack.ModuleManager
```

Comment:
Well, to me *Collection* will mean this is just a group on buildpacks, which is actually *true* but, it is more complex than that because it is a Collection of buildpacks organized according to some rules defined during the initialization of that Collection. That's why my first initial thought was to use the [Manager](https://www.eventhelix.com/design-patterns/manager/) or it could be a [facade](https://refactoring.guru/design-patterns/facade)

Reply:
yeah, `ManagedCollection` looks go to me

Reply:
Done